.circleci/config.yml

```diff
@@ -28,11 +28,11 @@ aliases:
 
```

Comment:
Appveyor need to change to.

Reply:
@gengjiawen change to what?

Reply:
I mean windows ci need related change too.

Reply:
@gengjiawen sorry, I have no experience of Appveyor. Please provide me exact change to make.

Reply:
https://github.com/facebook/react-native/blob/master/appveyor.yml, change ANDROID_BUILD_VERSION and ANDROID_TOOLS_VERSION is okay

Reply:
@gengjiawen done